Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -3962,7 +3962,7 @@ class WebPageProxy final : public API::ObjectImpl<API::Object::Type::Page>, publ
     bool m_waitingForContextMenuToShow { false };
 #endif
 
-    std::unique_ptr<WebsitePoliciesData> m_mainFrameWebsitePoliciesData;
+    RefPtr<API::WebsitePolicies> m_mainFrameWebsitePolicies;
```

Comment:
As we are only tracking main frame's `API::WebsitePolicies`, when WebKit clients sets different `API::WebsitePolicies` for subframes, who is keeping `WKUserContentController` or `WebUserContentControllerProxy` alive?